CHANGELOG.rst

```diff
@@ -12,18 +12,28 @@ Internal Changes
 
 * **Gateway**: Add endpoint for refreshing expiring Gitlab tokens
 * **Sessions**: Check LFS size and available disk space before cloning
+* **Knowledge Graph**: Improve functionality to refresh access tokens before expiration
+* **Knowledge Graph**: Enhance migration functionality to add missing CompositePlans when missing
```

Comment:
Would this work? (just trying to get rid of the double 'missing's- not a big deal)
```suggestion
* **Knowledge Graph**: Enhance migration functionality to restore missing CompositePlans 
```